companyweb_base/models/res_partner.py

```diff
@@ -0,0 +1,532 @@
+    cweb_url_enable = fields.Boolean(
+        "CompanyWeb Detailed Report Enabled", readonly=True
+    )
+    cweb_url_report = fields.Char("CompanyWeb Url Report", readonly=True)
+    cweb_url_report_enable = fields.Boolean(
+        "CompanyWeb Url Report Enabled", readonly=True
+    )
+    cweb_vat_liable = fields.Boolean("CompanyWeb Subject to VAT", readonly=True)
+    cweb_vat_liable_enable = fields.Boolean(
+        "CompanyWeb Subject to VAT Enabled", readonly=True
+    )
+    cweb_balance_data_enable = fields.Boolean(
+        "CompanyWeb Balance Data Enabled", readonly=True
+    )
+    cweb_balance_year = fields.Char("CompanyWeb Balance Year", readonly=True)
+    cweb_balance_year_visible = fields.Boolean(
+        "CompanyWeb Balance Year Visible", readonly=True
+    )
+    cweb_closed_date = fields.Date("CompanyWeb Closed Date", readonly=True)
+    cweb_closed_date_visible = fields.Boolean(
+        "CompanyWeb Closed Date Visible", readonly=True
+    )
+    cweb_equityCapital = fields.Float("CompanyWeb Equity Capital", readonly=True)
+    cweb_equityCapital_visible = fields.Boolean(
+        "CompanyWeb Equity Capital Visible", readonly=True
+    )
+    cweb_average_fte = fields.Float(
+        "CompanyWeb Average number of staff in FTE", readonly=True
+    )
+    cweb_average_fte_visible = fields.Boolean(
+        "CompanyWeb Average number of staff in FTE Visible", readonly=True
+    )
+    cweb_addedValue = fields.Float("CompanyWeb Gross Margin (+/-)", readonly=True)
+    cweb_addedValue_visible = fields.Boolean(
+        "CompanyWeb Gross Margin (+/-) Visible", readonly=True
+    )
+    cweb_turnover = fields.Float("CompanyWeb Turnover", readonly=True)
+    cweb_turnover_visible = fields.Boolean("CompanyWeb Turnover Visible", readonly=True)
+    cweb_result = fields.Float(
+        "CompanyWeb Fiscal Year Profit/Loss (+/-)", readonly=True
+    )
+    cweb_result_visible = fields.Boolean(
+        "CompanyWeb Fiscal Year Profit/Loss (+/-) Visible", readonly=True
+    )
+    cweb_prefLang = fields.Many2one(
+        "res.lang", string="CompanyWeb Preferred Language", readonly=True
+    )
+    cweb_prefLang_enable = fields.Boolean(
+        "CompanyWeb Preferred Language Enabled", readonly=True
+    )
+
+    cweb_show_button_enhance = fields.Boolean(
+        "CompanyWeb Button Enhance Enabled", compute="_compute_cweb_show_button_enhance"
+    )
+    cweb_show_button_address = fields.Boolean(
+        "CompanyWeb Button Address Enabled", compute="_compute_cweb_show_button_address"
+    )
+    cweb_show_tab = fields.Boolean(
+        "CompanyWeb Tab Enabled", compute="_compute_cweb_cweb_show_tab"
+    )
+
+    @api.depends("cweb_image")
+    def _compute_cweb_image(self):
+        """solution found for email is to compute yhe html
+        i'll store the full html tag"""
+        for rec in self:
+            img_url = "/companyweb_base/static/img/cweb_barometer/%s" % rec.cweb_image
```

Comment:
Here you can user `odoo.modules.module.rec.get_resource_path(''companywev_base", "static", "img", "cweb_barometer", rec.cweb_image)` to chec that the file exists. If it does, return the img_url as you do here, otherwise return an empty html or some warning that the image was not found locally. This should also serve as a good protection from html injection in cweb_image.